src/app/charts/core/files/secret/tpl/arkcase-server.yaml

```diff
@@ -74,6 +74,11 @@ fileManagement:
     secretKey: "${ARKCASE_CONTENT_PASSWORD}"
 {{- end }}
 
+{{- if (eq "box" $content.dialect) }}
+box:
+  privateKeyPath: "${CONTENT_MAIN_PRIVATE_KEY_PATH:/app/home/.content-private-key}"
```

Comment:
The variable is never defined. In this specific scenario, it's better to leave it hardcoded since this is the path that will be rendered every time and we won't be messing with this .

Alternatively, you can use `cminfo` to pull in the value defined there (which I like better since it gives us a bit more freedom).